backend/app/main.py

```diff
@@ -3,18 +3,18 @@
 import argparse
 
 from uvicorn.config import LOGGING_CONFIG
-from starlette.graphql import GraphQLApp
+# from starlette.graphql import GraphQLApp
```

Comment:
These commented out lines can be deleted if you are sure we don't need them. This one we for sure don't need as it was from when I was playing with getting GraphQL working but realized it was too much to take on at the beginning at least.